code/modules/cargo/bounties/chef.dm

```diff
@@ -0,0 +1,161 @@
+	wanted_types = list(
+		/obj/item/weapon/reagent_containers/food/snacks/variable/cake,
+		/obj/item/weapon/reagent_containers/food/snacks/sliceable/cake)
+
+/datum/bounty/item/chef/soup/New()
+	..()
+	name = "Soup"
+	description = "To quell the homeless uprising, [current_map.company_name] will be serving soup to all underpaid workers. Ship any type of soup."
+	reward = 3000
+	required_count = 3
+	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/soup)
+
+/datum/bounty/item/chef/popcorn/New()
+	..()
+	name = "Popcorn Bags"
+	description = "Upper management wants to host a movie night. Ship bags of popcorn for the occasion."
+	reward = 3000
+	required_count = 3
+	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/popcorn)
+
+/datum/bounty/item/chef/icecreamsandwich/New()
+	..()
+	name = "Ice Cream Sandwiches"
+	description = "The Air Conditioning System in the CCIA Offices has failed. Ship some icecream."
+	reward = 4000
+	required_count = 3
+	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/icecreamsandwich)
+/*
+/datum/bounty/item/chef/bread/New()
+	..()
+	name = "Bread"
+	description = "Problems with central planning have led to bread prices skyrocketing. Ship some bread to ease tensions."
+	reward = 1000
+	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/store/bread, /obj/item/weapon/reagent_containers/food/snacks/breadslice, /obj/item/weapon/reagent_containers/food/snacks/bun, /obj/item/weapon/reagent_containers/food/snacks/pizzabread, /obj/item/weapon/reagent_containers/food/snacks/rawpastrybase) 
+*/
+/datum/bounty/item/chef/pie/New()
+	..()
+	name = "Pie"
+	description = "3.14159? No! [current_map.boss_short] management wants edible pie! Ship a whole one."
+	reward = 3142
+	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/pie)
+
+/datum/bounty/item/chef/salad/New()
+	..()
+	name = "Salad or Rice Bowls"
+	description = "[current_map.boss_short] management is going on a health binge. Your order is to ship salad or rice bowls."
+	reward = 3000
+	required_count = 3
+	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/salad)
+
+/datum/bounty/item/chef/carrotfries/New()
+	..()
+	name = "Carrot Fries"
+	description = "Night sight can mean life or death! A shipment of carrot fries is the order."
+	reward = 3500
+	required_count = 3
+	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/carrotfries)
+
+/datum/bounty/item/chef/superbite/New()
+	..()
+	name = "Super Bite Burger"
+	description = "Commander Tubbs thinks he can set a competitive eating world record. All he needs is a super bite burger shipped to him."
+	reward = 12000
+	wanted_types = list(/obj/item/weapon/reagent_containers/food/snacks/burger/superbite)
+
+/datum/bounty/item/chef/poppypretzel/New()
+	..()
+	name = "Poppy Pretzel"
+	description = "[current_map.boss_name] needs a reason to fire their HR head. Send over a poppy pretzel to force a failed drug test."
```

Comment:
NanoTrasen wouldn't do this, at least publicly.

Reply:
Changed the wording